docs/coa-aks.md

```diff
@@ -0,0 +1,53 @@
+## Cromwell on AKS Instructions and Troubleshooting
+
+### Deployment Dependencies
+The CoA deployer requires the user to have Helm 3 installed locally to deploy with AKS. Use the flag "--HelmExePath HELM_PATH" to give the deployer the path to the helm exe, if no flag is passed the deployer will assume Helm is installed with Chocolatey at "C:\\ProgramData\\chocolatey\\bin\\helm.exe".
+
+### Deployment Models
+
+- ### CoA provisioned AKS account
+    Add the flag "--UseAks true" and the deployer will provision an AKS account and run its containers in AKS rather than provisioning a VM. 
+- ### Shared AKS account with CoA namespace
+    Add the flags "--UseAks true --AksClusterName {existingClusterName}", where the user has admin access to the existing AKS account, the deployer will deploy blob-csi-driver, and aad-pod-identity to the kube-system namespace, and then deploy CoA to the namespace "coa". Add the flag "--AksCoANamespace {namespace}" to override the default namespace.
```

Comment:
What specific Azure role does the user need to have, that meets the principle of least privilege?

Reply:
Updated with specific roles, "Contributor" or "Azure Kubernetes Service Contributor"